pyproject.toml

```diff
@@ -45,7 +45,7 @@ app = "app.__main__:run_cli"
 
 [build-system]
 build-backend = "hatchling.build"
-requires = ["hatchling", "nodeenv"]
+requires = ["hatchling", "nodeenv", "setuptools"]
```

Comment:
why setuptools?

Reply:
Actually, this can be removed.  This was to debug an issue in my local env.

Reply:
```suggestion
requires = ["hatchling", "nodeenv"]
```